rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.82.0"
+targets = ["wasm32-unknown-unknown"]
```

Comment:
It seems that github action correctly sets provided 1.82 version, but cross compilation for wasm32 target is not done.  

Is targets line needed for this library? If so, how wasm build should be correctly tested inside github Action ?

Reply:
@PPakalns the `targets` value can simply be deleted in fact - I thought that perhaps you would like to have that target installed by default in this project (in addition to whatever target-triple rustup will automatically pick for your host system) because it's the target that `egui` specifies for web builds. I can update the PR to remove the `target` though to keep things simple :)